### DISCUSSION TITLE
Example Content Calculation Error

### DISCUSSION THREAD

**Discussion by TullyMonster** [biostars/biostar-handbook-issues#294] [General]
https://www.biostarhandbook.com/survival-guide.html#:~:text=Produces%20the%20effect%20size%3A
The result is 436.5 instead of 453, which seems.

**Comment by ialbert** [comment]
ahem, indeed, sorry about that,

I think I must have played around with the numbers for some other reasons and did not update all fields

apologies, will be fixed soon

**Comment by ialbert** [comment]
The issue has been corrected.
